docs/docs/code/nodejs/README.md

```diff
@@ -133,14 +133,16 @@ For example, let's define a `firstName` prop. This will allow us to freely enter
 export default defineComponent({
   props: {
     firstName: {
-      type: 'string',
-      label: 'Your first name',
-      default: 'Dylan',
-    }
+      type: "string",
+      label: "Your first name",
+      default: "Dylan",
+    },
   },
   async run({ steps, $ }) {
-    console.log(`Hello ${this.firstName}, congrats on crafting your first prop!`);
-  }
+    console.log(
+      `Hello ${this.firstName}, congrats on crafting your first prop!`
```

Comment:
```suggestion
      `Hello ${this.firstName}, congrats on crafting your first prop!`
```

Reply:
Changed "congratulations" to "congrats" to follow Dreyer's English on using shorter words